src/main.rs

```diff
@@ -308,10 +309,15 @@ fn main() {
         SubCommand::with_name( "deploy" )
             .about( "Deploys your project so that its ready to be served statically" );
 
+    let mut prepare_emscripten_subcommand =
+        SubCommand::with_name( "prepare-emscripten" )
+            .about( "Prepares and installs emscripten" );
+
     build_subcommand = add_shared_build_params( build_subcommand );
     test_subcommand = add_shared_build_params( test_subcommand );
     start_subcommand = add_shared_build_params( start_subcommand );
     deploy_subcommand = add_shared_build_params( deploy_subcommand );
+    prepare_emscripten_subcommand = add_shared_build_params( prepare_emscripten_subcommand );
```

Comment:
This shouldn't have any shared build params attached to it as it doesn't use any of them anyway.